jina/hubble/hubapi.py

```diff
@@ -76,7 +77,7 @@ def get_secret_path(inode: str) -> 'Path':
 def load_secret(work_path: 'Path') -> Tuple[str, str]:
     """Get the UUID and Secret from local
     :param work_path: the work path of the executor
-    :return: the UUID and secret
+    :return: the UUID and secret and task_id
```

Comment:
Why save task_id to the local secret cache file?

Reply:
mm because we can't get the executor id when we first create the executor. The executor id can only be obtained after passing the query.   if we don't get the executor id, we can't  update  the executor